test/e2e_node/util.go

```diff
@@ -94,6 +94,19 @@ var (
 	kubeletCfg *kubeletconfig.KubeletConfiguration
 )
 
+func initFeatureGates(featureGates map[string]bool) {
+	err := utilfeature.DefaultMutableFeatureGate.SetFromMap(featureGates)
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "ERROR: initialize feature gates: %v", err)
+		os.Exit(1)
+	}
+}
+
+func isFeatureGateEnabled(feature featuregate.Feature) bool {
```

Comment:
let's add a comment saying that those are feature gates set at test start. I vagualy remember people was trying to add tests that will update feature gates and restart kubelet to validate the use case when feature gate is being disabled